.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Question
+about: Ask a question about design here. Ask about usage on https://scanpy.discourse.group/
```

Comment:
This is good, but could use a bit of clarification. Maybe add a line in the template on:

> Is your question "how can I do X/Y/Z?", then please ask this in  `https://scanpy.discourse.group/`